Yurii_Khomych/l_4_iterators_generators/HW_4_yuliia/Generators/generators.py

```diff
@@ -0,0 +1,27 @@
+# generator example 1
+def example_1():
+    my_list = []
+    for x in range(10):
+        my_list.append(x)
+    return my_list
+
+
+e = example_1()
+print(e)
+
+# generator statement example
+e3 = [num*4 for num in range(5)]
```

Comment:
і це теж)

Reply:
Corrected here https://github.com/YuriiKhomych/ITEA-advanced/pull/11 because of incorrect parent branch 
I'll remove this PR and branch as soon as https://github.com/YuriiKhomych/ITEA-advanced/pull/11 will be checked